python/grass/script/tests/grass_script_setup_test.py

```diff
@@ -8,10 +8,8 @@
 import grass.script as gs
 
 
-# All init tests change the global environment, but when it really matters,
-# we use a separate process.
-# Ideally, the functions would support env parameter and the test
-# would mostly use that.
+# This is useful when we want to ensure that function like init does
+# not change the global environment.
 def run_in_subprocess(function):
```

Comment:
Oh, that code is duplicated? It isn't shared between tests? (It's not this PR's fault though 😉)

Reply:
Right, two things would be useful for reducing that duplication: a common approach to shared code for pytest-based tests and an general solution to this subprocess/ctypes issue (FHS will help with the availability of libraries. Exits and crashes from C code need different solution, but perhaps pytest can handle that?).